### PR TITLE
laa: added N_from_Wr() and N_from_Wr_transition() with tests.

### PR DESCRIPTION
**Pull request by ntejos**
N_from_Wr() and N_from_Wr_transition() added to linetools.analysis.absline; these are useful for getting upper limits on N from measured Wr upper limits. Assume tau0<<1 where the relation between N and Wr is independent on Doppler parameter and gamma.

### CODE REVIEW

File: linetools/analysis/absline.py [hunk 1]
```diff
@@ -531,3 +567,47 @@ def Wr_from_N_transition(N, transition, llist='ISM'):
     # return
     return Wr_from_N(N, wrest, fosc)
 
+
+def N_from_Wr_transition(Wr, transition, llist='ISM'):
```
Comment (profxj) [inline linetools/analysis/absline.py]: While over-loading input statements is a bit dangerous, I might
prefer it here, i.e. combine these 2 methods and the 2nd input
can either be a Quantity (wrest) or a str (transition).  Could even
allow an AbsLine.

Comment (ntejos) [inline linetools/analysis/absline.py]: I would rather not overload this the input statement as it goes somewhat against my preferred direction when coding. Instead, I have added the corresponding method to the AbsLine class, so you can use it directly from there. I hope you agree with this decision.

**Comment by ntejos** [comment]
Have also added some basic documentation in AbsLine.rst regarding these methods.

**Comment by profxj** [comment]
Looks good.  Please merge

**Pull request merged**